### PR TITLE
Add no responders

### PR DESCRIPTION
**Pull request by Jarema**

### CODE REVIEW

File: async-nats/src/connection.rs [hunk 1]
```diff
@@ -273,6 +264,8 @@ impl Connection {
                         subject,
                         headers: Some(headers),
                         payload,
+                        status: maybe_status,
+                        description: maybe_description,
```
Comment (caspervonb) [inline async-nats/src/connection.rs:268]: Do we ever have a payload and description?

File: async-nats/src/lib.rs [hunk 2]
```diff
@@ -1068,17 +1077,17 @@ pub struct Message {
     pub reply: Option<String>,
     pub payload: Bytes,
     pub headers: Option<HeaderMap>,
+    status: Option<NonZeroU16>,
+    description: Option<String>,
 }
 
 impl Message {
     fn is_no_responders(&self) -> bool {
```
Comment (caspervonb) [inline async-nats/src/lib.rs:1085]: Make this an extension trait?

```suggestion
    fn is_no_responders(&self) -> bool {
```

Comment (Jarema) [reply async-nats/src/lib.rs:1085]: it is a private function now.

**Review by caspervonb — APPROVED** [review]
lgtm

File: nats/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# 0.20.1
+## Overview
+A patch release fixing stack overflow happening when there was kv watcher for a key that did not change for a long time.
```
Comment (caspervonb) [inline nats/CHANGELOG.md:3]: Watch out for this bit getting squashed.

**Pull request merged**